### PR TITLE
Widgets\Drawer - add Vue approach to each snippet

### PR DESCRIPTION
**Pull request by IgnatovDan**
See these snippets applications in action:

- Getting started 
  - jQuery - https://codesandbox.io/s/dxdrawer-jquery-with-router-82dkm
  - Angular - https://codesandbox.io/s/dxdrawer-angular-router-3hzmd
  - React - https://codesandbox.io/s/dxdrawer-reactrouter-ugedo
  - Vue - https://codesandbox.io/s/dxdrawer-with-vuerouter-2syjp
 - Create the Drawer (Vue) - https://codesandbox.io/s/dxdrawer-create-drawer-sy1fh
 - Open and Close the Drawer (Vue) - https://codesandbox.io/s/dxdrawer-createdrawer-lw8bk
 - Implement Navigation (Vue) - https://codesandbox.io/s/dxdrawer-with-vuerouter-2syjp
 - Configure the Reveal Behavior (Vue) - https://codesandbox.io/s/dxdrawer-createdrawer-lw8bk
 - Configure Interaction with the View (Vue) - https://codesandbox.io/s/dxdrawer-openedstatemode-6o7u1

### CODE REVIEW

File: concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/00 Create the Drawer.md [hunk 1]
```diff
@@ -173,6 +173,30 @@ In addition, you can specify the [minSize](/api-reference/10%20UI%20Widgets/dxDr
     }
     export default App;
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+    <div>
```
Comment (EugeniyKiyashko) [inline concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/00 Create the Drawer.md]: It seems that "div" wrap is not neccessary

File: concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/00 Create the Drawer.md [hunk 2]
```diff
@@ -173,6 +173,30 @@ In addition, you can specify the [minSize](/api-reference/10%20UI%20Widgets/dxDr
...
+        <DxDrawer opened="true" minSize="37" height="250" template="template" reveal-mode="expand">
```
Comment (EugeniyKiyashko) [inline concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/00 Create the Drawer.md]: minSize="37" -> :minSize="37"; height -> :height

File: concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/15 Implement Navigation.md [hunk 3]
```diff
@@ -401,6 +401,128 @@ In this tutorial, we use the **List**:
...
+            loadView(e) {
+                this.$router.push(e.addedItems[0].filePath);
+            }
+        }
+    };
+    </script>
+
+    <!-- tab: Inbox.vue -->
+    <template>
+    <div>Inbox</div>
```
Comment (EugeniyKiyashko) [inline concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/15 Implement Navigation.md]:
```suggestion
        <div>Inbox</div>
```

File: concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/20 Configure the Reveal Behavior.md [hunk 4]
```diff
@@ -44,6 +44,17 @@ When you open the **Drawer**, it can slide in or expand from the closed position
...
     export default DxComponent;
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <div>
+            <DxDrawer ...
+                reveal-mode="expand" >
+            </DxDrawer>
```
Comment (EugeniyKiyashko) [inline concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/20 Configure the Reveal Behavior.md]: Replace with self-close tag />

File: concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/00 Create the Drawer.md (hunk 1 above)
Comment (IgnatovDan) [reply concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/00 Create the Drawer.md]: There should be 1 root element in a template: https://github.com/vuejs/vue/issues/7088 - Allow more than 1 root element for Template

So, we create it in our demos:
https://js.devexpress.com/Demos/WidgetsGallery/Demo/Drawer/LeftOrRightPosition/Vue/Light/
<template>
  <div>
    <DxToolbar :items="toolbarContent"/>

https://js.devexpress.com/Demos/WidgetsGallery/Demo/Accordion/Overview/Vue/Light/
<template>
  <div id="accordion">
    <DxAccordion

And, it's very easy to copy-paste our code snippet to another project and add a new markup without wrapping all into a '1 root element': explicit div will help avoid some cases of this stupid mistake.

File: concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/15 Implement Navigation.md (hunk 3 above)
Comment (IgnatovDan) [reply concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/15 Implement Navigation.md]: https://github.com/DevExpress/devextreme-documentation/pull/447/commits/3a48ba66c708d9aefd47645f68939ae5ae89b28c

File: concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/20 Configure the Reveal Behavior.md (hunk 4 above)
Comment (IgnatovDan) [reply concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/20 Configure the Reveal Behavior.md]: https://github.com/DevExpress/devextreme-documentation/pull/447/commits/3b6e2a9ca3d72f82ec1c8e1a078419f0f8d830c5

File: concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/00 Create the Drawer.md (hunk 2 above)
Comment (IgnatovDan) [reply concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/00 Create the Drawer.md]: https://github.com/DevExpress/devextreme-documentation/pull/447/commits/468802ad7b32c43ebfbc21337c9b5557fee71446

File: concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/00 Create the Drawer.md (hunk 1 above)
Comment (EugeniyKiyashko) [reply concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/00 Create the Drawer.md]: Yes, exactly.

Comment (IgnatovDan) [reply concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/00 Create the Drawer.md]: https://github.com/DevExpress/devextreme-documentation/pull/447/commits/6ca95dbc559ad2a14c7775de3e636da197ebbbc1

File: concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/00 Create the Drawer.md (hunk 2 above)
Comment (IgnatovDan) [reply concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/00 Create the Drawer.md]: https://github.com/DevExpress/devextreme-documentation/pull/447/commits/73c432f42346324ac4d7bd904c687617cbdfe760

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by novsstation — APPROVED** [review]

**Review by jsdmitry — APPROVED** [review]

**Review by LexDX — APPROVED** [review]

**Comment by RomanTsukanov** [comment]
I've opened PR #481 which not only contains slightly reworked Vue code snippets from this PR but also addresses [issues](https://supportcenter.devexpress.com/internal/ticket/details/T867807) in the code for other frameworks. I'm closing this PR.

@IgnatovDan, thanks for the Vue snippets.

**Pull request closed without merging**